ambry-store/src/main/java/com.github.ambry.store/BlobStore.java

```diff
@@ -360,6 +360,31 @@ public long getSizeInBytes() {
     return index.getLogUsedCapacity();
   }
 
+  /**
+   * Return total capacity of the {@link BlobStore} in bytes
+   * @return the total capacity of the {@link BlobStore} in bytes
+   */
+  long getCapacityInBytes() {
+    return capacityInBytes;
+  }
+
+  /**
+   * Return the {@link Time} instance being used
+   * @return the {@link Time} instance being used
+   */
+  Time getTime() {
+    return time;
+  }
+
+  /**
+   * Fetches a list of {@link LogSegment} names which has entries outside of {@link Journal}
+   * @return list of {@link LogSegment} names which has entries outside of {@link Journal}
```

Comment:
define what happens if there are no log segments outside the journal. 0 sized list or null?